specs/app.spec.js

```diff
@@ -1,2 +1,80 @@
 // eslint-disable-next-line no-unused-vars
-import { nameIsValid, fullTrim, getTotal, sum } from "../src/app.js";
+import { nameIsValid, fullTrim, getTotal } from "../src/app.js";
+
+describe('Tests for function nameIsValid', () => {
+test ('имя на кириллице не проходит', () => {
+expect(nameIsValid('Тома')).toBe(false);
+});
+
+test ('имя правильного формата проходит', () => {
+  expect(nameIsValid('vova')).toBe(true);
+  });
+
+test ('имя, состоящее из одного символа, не проходит', () => {
+  expect(nameIsValid('v')).not.toBe(true);
+  });
+
+test ('имя, состоящее из двух букв, проходит ', () => {
+  expect(nameIsValid('li')).toBe(true);
+  });
+
+test ('если имя не определено, оно не проходит ', () => {
+  expect(nameIsValid()).toBe(false);
+  });
+
+test ('имя не является null', () => {
+  expect(nameIsValid('Toma')).not.toBeNull;
```

Comment:
вы забыли добавить `()` в конце

```suggestion
  expect(nameIsValid('Toma')).not.toBeNull();
```